packages/cli/src/__tests__/ceramic-logger-plugins.test.ts

```diff
@@ -10,105 +10,181 @@ describe('Ceramic Logger Plugins', () => {
 
     const yesterday = new Date('2020-10-30')
     const today = new Date('2020-10-31')
+    const inconclusive = new Date(0)
+
+    const YESTERDAY = 0
+    const TODAY = 1
+    const LARGE = 2
+    const INCONCLUSIVE = 3
+    const BLOCKING = 4
 
     describe('LogToFiles', () => {
         let mockFiles: Array<any>
         let mockFs: any
+        let blockedFiles: BlockedFiles
+        let basePath: string
 
         beforeEach(async () => {
+            basePath = '/usr/local/var/log/'
```

Comment:
Does this work on Windows?

Reply:
i have to change this path anyway because it doesn't work with docker anymore. Will look into something that should work independent of os